src/hello_client.erl

```diff
@@ -165,12 +166,16 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{waiting_for_pong = true, keep_alive_interval = KeepAliveInterval}) ->
+    ?LOG_ERROR("There is no PONG answer on PING for ~p msec. Client will be restarted.", [KeepAliveInterval]),
```

Comment:
so, a user of a tiny tiny tiny little hello-service would need the hello-service and then grafana (or the like) to get the very very tiny information for how long the other service did not answer?

really? is it such a hard problem to remember the last time the hello service has received an answer for a pong? we are talking about one 64bit value, at most.